speech/setup.py

```diff
@@ -50,14 +50,14 @@
 }
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.24.0, < 0.25dev',
+    'google-cloud-core >= 0.24.0, < 0.26dev',
```

Comment:
Oh, I wanted to talk to you about that. Will discuss offline. (That is an intentional decision, not a mistake, but I want to run it by you first. :-))